tickets/src/components/content/create/CreateEventForm.jsx

```diff
@@ -0,0 +1,194 @@
+import PropTypes from 'prop-types'
```

Comment:
This is a new conmponent (most of it taken from CreateEvent) to split the purpose of selecting the lock vs editing the event.